src/core/meshcop/network_name.hpp

```diff
@@ -260,6 +260,14 @@ class NetworkNameManager : public InstanceLocator, private NonCopyable
      *
      */
     Error SetDomainName(const NameData &aNameData);
+
+    /**
+     * Checks whether the Thread Domain Name is currently set to the default name.
+     *
+     * @returns true if Thread Domain Name equals "DefaultDomain", false otherwise.
+     */
```

Comment:
```suggestion
     * @returns true if Thread Domain Name equals "DefaultDomain", false otherwise.
     * 
     */
```

Reply:
This no longer applies now that #10748 is merged.